### ISSUE TITLE
Applying inverse DVF gives blurry results

### ISSUE THREAD

**Issue by burrusa** [voxelmorph/voxelmorph#610]
**Task** I am registering 3D brain MRI data

**What have you tried**

Hi, thank you again for your tool which I find great! 
I am currently using the TensorFlow version to register 3D brain MRI data and in my application, I need to use the inverse deformation field. When I apply it I get blurry results, I tried the Pytorch version and I have the same problem...

Here is how I call the model and how I apply the transforms:

config = dict(inshape=inshape, input_model=None)

    dvf = vxm.networks.VxmDense.load(PATH_MODEL, bidir=True, **config).register(normalized_moving_array, normalized_fixed_array)

    moved = vxm.networks.Transform(inshape, nb_feats=nb_feats).predict([normalized_moving_array, dvf[0]])

    moved_back=vxm.networks.Transform(inshape, nb_feats=nb_feats).predict([moved, dvf[1]])
    
    My warped image (moved) looks great, however, when I apply the inverse deformation field on it (moved_back) I get a blurrier version of my original moving image. I added images so you can visualize better.

<img width="377" alt="Screenshot 2024-07-19 at 16 45 33" src="https://github.com/user-attachments/assets/fabc4909-0083-45cf-9464-951fa9c75ef8">

<img width="377" alt="Screenshot 2024-07-19 at 16 45 37" src="https://github.com/user-attachments/assets/e3b6117a-bd40-4129-9829-db81bc37e8e2">

Is there a reason why this is happening ? Do you see a way how I could solve this problem?

Thank you very much for your help,
Anaïs
